src/native/tls_connection_options.c

```diff
@@ -25,21 +25,51 @@ JNIEXPORT
 jlong JNICALL Java_software_amazon_awssdk_crt_io_TlsConnectionOptions_tlsConnectionOptionsNew(
```

Comment:
whoa check out this C file just sitting around for years, never getting used, but now we are!